.prettierignore

```diff
@@ -1,3 +1,4 @@
 **/fixtures/**
 **/fixtures-go/**
 /docs/vendor/**
+/internal/output/html/*
```

Comment:
fwiw it looks like you should be able to format at least the css and js files with Prettier as they've not got any templating, which I think would be good for consistency

You've also not added any final newlines which is something Prettier would catch if we can run it on these files 😅

(that is defined in our `.editorconfig` - if you're using vscode might be worth checking you've got https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig installed and enabled)

Reply:
Thanks Gareth! I actually didn't install editorConfig. Updated HTML with newlines and only disable linting on HTML templates

Reply:
@G-Rath I'm merging this PR now to gather internal feedback. Feel free to try out the new HTML output and share any issues or ideas you have here: https://github.com/google/osv-scanner/issues/1274